.gitignore

```diff
@@ -1 +1,2 @@
-cmd/
\ No newline at end of file
+cmd/
+.env
```

Comment:
With respect to the above comment...if you need to do some testing that requires loading environment variables from an env file, please create a `cmd` folder and perform the testing inside that folder. 

Your test cases might need to use some secret values....so, configure them to be gotten from the env.

`PinataJWT := os.Getenv("PINATA_JWT")`
Then manually export this env into your environment rather than use an env file. 

In your terminal
`export PINATA_JWT="somevalues"`
or
`set PINATA_JWT="somevalues"`

But if you still want to load things from an env file, then those should be done inside of the cmd folder so they don't get pushed into the commit history.